### PR TITLE
TNO-1388: Add ability for subscriber to transcribe

### PR DESCRIPTION
**Pull request by jdtoombs**
This PR includes the functionality that allows subscribers to transcribe a video if it has not been transcribed already. Clearly my own instance of Azure is having some issues as seen by the text it has generated..

However, you can see that after the transcription is appended to the body the application will add a new section for the transcription as was discussed with Bobbi on Friday.

image

### CODE REVIEW

File: app/subscriber/src/features/content/view-content/ViewContent.tsx [hunk 1]
```diff
@@ -153,7 +188,33 @@ export const ViewContent: React.FC = () => {
         >
           <p>{parse(content?.summary ?? '')}</p>
         </Show>
+        <Show visible={content?.contentType === ContentTypeName.Snippet}>
+          <Button
+            onClick={() => handleTranscribe()}
+            variant={ButtonVariant.action}
+            className="transcribe-button"
+            disabled={
+              (!!content?.fileReferences && !content?.fileReferences.length) ||
```
Comment (jdtoombs) [inline app/subscriber/src/features/content/view-content/ViewContent.tsx:197]: Button disabled when there is no video attached to a snippet, or if the transcription has already been completed,

File: app/subscriber/src/features/content/view-content/utils/WorkOrderStatus.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import { FaCheckCircle, FaExclamationCircle, FaTimesCircle } from 'react-icons/fa';
```
Comment (jdtoombs) [inline app/subscriber/src/features/content/view-content/utils/WorkOrderStatus.tsx:1]: Should I just add this to tno-core?

Comment (jdtoombs) [reply app/subscriber/src/features/content/view-content/utils/WorkOrderStatus.tsx:1]: And other related items

Comment (Fosol) [reply app/subscriber/src/features/content/view-content/utils/WorkOrderStatus.tsx:1]: For now I don't think so.

**Review by Fosol — APPROVED** [review]

**Pull request merged**